core/mono-samples/iOS/Program.csproj

```diff
@@ -0,0 +1,55 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <RuntimeIdentifier>ios-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Runtime.iOS.Sample.Mono" Version="6.0.0-dev" />
+  </ItemGroup>
+
+  <UsingTask TaskName="AppleAppBuilderTask"
+             AssemblyFile="$(NuGetPackageRoot)microsoft.net.runtime.ios.sample.mono/6.0.0-dev/tools/net6.0/AppleAppBuilder.dll" />
+
+  <Target Name="BuildAppBundle" AfterTargets="CopyFilesToPublishDirectory;Publish;BuildAppleAppBuilder">
```

Comment:
I don't see the `BuildAppleAppBuilder`  target anywhere, is it needed?
Additionally, I think we only need to run after the Publish target, not CopyFilesToPublishDirectory.

Reply:
Ah right, removing both.